shell/src/main/java/alluxio/cli/fs/command/GetFaclCommand.java

```diff
@@ -51,6 +52,12 @@ protected void runPlainPath(AlluxioURI path, CommandLine cl)
     for (String entry : status.getAclEntries()) {
       System.out.println(entry);
     }
+    List<String> defaultAclEntries = status.getDefaultAclEntries();
```

Comment:
this should never be `null`, right? It will just be an empty string?

Reply:
right, removed because it is not necessary